test/autoload/test_autoload.py

```diff
@@ -0,0 +1,38 @@
+# Owner(s): ["module: unknown"]
+
+import os
+import sys
+
+import torch
+
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestAutoload(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.device_backend_path = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), "device_backend")
+        )
+
+        # add the extension to the system path
+        if self.device_backend_path not in sys.path:
+            sys.path.insert(0, self.device_backend_path)
+
+    def test_autoload(self):
+        # after importing the extension, the value of this environment variable should be true
+        torch.import_device_backends()
+        value = os.getenv("IS_CUSTOM_DEVICE_BACKEND_IMPORTED", "false")
```

Comment:
Besides checking the environment variable, are you going to add test to autoload an extension (maybe a mock one)?

Reply:
Thanks for your review. Please have a look at the code I just updated.